src/docfx.website.themes/default/styles/docfx.js

```diff
@@ -20,7 +20,10 @@ $(function () {
   renderFooter();
   renderLogo();
 
-  window.refresh = function () {
+  window.refresh = function (rawTitle, content) {
```

Comment:
>rawTitle, content [](start = 29, length = 17)

looks like this is specialized for conceptual topics, what about mref and rest? It does not sound like a general solution in the long run?